### ISSUE TITLE
Floating point exception (core dumped) Arch32

### ISSUE THREAD

**Issue by coreybruce** [ruffle-rs/ruffle#7196]
### Describe the bug

Installed Ruffle-git off the AUR which compiled ruffle for Linux 32bit and run Hobo 3 which I know works on Ruffle and Linux x64 but when I run it on Arch32 it will fail with a error message ```Floating point exception (core dumped)```

### Expected behavior

Game runs (not sure if this is a limitation of the 32bit laptop or a bug on ruffle)

### Affected platform

Desktop app

### Operating system

Arch 32

**Comment by n0samu** [comment]
Can you run `ruffle --version` and verify that you're using the latest version of Ruffle?

**Comment by coreybruce** [comment]
Yeah sure 

```
[corey@Corey-Fujitsu ~]$ ruffle --version
Ruffle 0.1.0-nightly (487017e7c 2022-06-07)
```

**Comment by coreybruce** [comment]
Is this correct? And do you know what's happening?

**Comment by n0samu** [comment]
Yes it's the right version, I just wanted to make sure. Sorry but I don't think any of us have a 32-bit Linux system to test with so we're not sure what's going on.

**Comment by coreybruce** [comment]
> Yes it's the right version, I just wanted to make sure. Sorry but I don't think any of us have a 32-bit Linux system to test with so we're not sure what's going on.

Ahh ok it's always good to check. Oh I see, could you do a VM to test than?

**Comment by n0samu** [comment]
Ruffle desktop builds require hardware acceleration, so I'm not sure a VM would help

**Comment by coreybruce** [comment]
Could you do a gpu passthrough also even tho it needs GPU acceleration to run properly? If you could still do it without you acceleration would you from there be able to see what's wrong? It wouldn't matter how the game runs as the main issue is just getting it to work properly

**Comment by n0samu** [comment]
I don't think I have the right knowledge to figure this out but hopefully one of the developers who works with the project's Rust code can give it a quick look in a VM at least.

**Comment by coreybruce** [comment]
Damn ok thank you for the help

**Comment by Aaron1011** [comment]
Are you able to provide a backtrace or core dump?

**Comment by coreybruce** [comment]
Sure just tell me what I need to do and I'll provide the logs

**Comment by coreybruce** [comment]
Hello?

**Comment by Aaron1011** [comment]
Sorry for the delay. Does a core dump file get generated when the exception occurs? If not, can you try running Ruffle under `gdb` and running `bt` after the exception occurs?

**Comment by coreybruce** [comment]
> Sorry for the delay. Does a core dump file get generated when the exception occurs? If not, can you try running Ruffle under `gdb` and running `bt` after the exception occurs?

No it doesn't, so how do I use gdb?

**Comment by n0samu** [comment]
Take a look at https://wiki.archlinux.org/title/Core_dump

**Comment by coreybruce** [comment]
Having a bit of issues using it, could you please walk me through what I need to do @n0samu

**Comment by n0samu** [comment]
I'm not experienced with Linux so I can't help much, I was just pointing you to the best guide I could find

**Comment by coreybruce** [comment]
I have a dump file but I have no way to actually read it

**Comment by n0samu** [comment]
Please upload it somewhere for us to take a look at.

**Comment by coreybruce** [comment]
I figured out that it's defaulting to vulkan which is causing the issue, I tried using opengl but it's saying it doesn't have a compatible graphics device

**Comment by coreybruce** [comment]
Could anyone help with this issue please

**Comment by n0samu** [comment]
I think you should go to the Discussions tab or join our Discord server, and explain exactly what you're trying to do in one place. Then we can try to figure out whether your goals are feasible. It seems like you're trying to build and run Ruffle for several different Raspberry Pi devices but I think a bit more detail and insight into your overall goals would help a lot.

**Comment by coreybruce** [comment]
Sure I could , I am trying to do is run ruffle on a Raspberry pi 4 and a 32bit Fujitsu laptop running Arch32 and also have it working on old 32bit Linux machines and mainly Arm64 for added support
